src/Socket_p.h

```diff
@@ -362,7 +359,7 @@ namespace Arcus
             return;
         }
 
-        uint32_t message_size = message->ByteSize();
+        uint32_t message_size = message->ByteSizeLong();
```

Comment:
This function was introduced in Protobuf 3.1. It doesn't exist yet in Protobuf 3.0.0. Our CI is complaining about an unknown function because of this.